From 9cc5bca1394e714672f175855a8f4c02bef68ee8 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sun, 4 Apr 2021 17:20:06 +0200
Subject: [PATCH] Use 'unsigned int' instead of 'uint'

The latter isn't defined on musl libc.

In file included from /home/pmos/build/src/qtmir-824e8b5bf92c2374d331af99e6845ea594ec0ea5/src/platforms/mirserver/miral/persist_display_config.cpp:21:
/home/pmos/build/src/qtmir-824e8b5bf92c2374d331af99e6845ea594ec0ea5/include/qtmir/miral/display_configuration_storage.h:35:25: error: 'uint' was not declared in this scope; did you mean 'int'?
   35 |     mir::optional_value<uint> clone_output_index;
      |                         ^~~~
      |                         int
/home/pmos/build/src/qtmir-824e8b5bf92c2374d331af99e6845ea594ec0ea5/include/qtmir/miral/display_configuration_storage.h:35:29: error: template argument 1 is invalid
   35 |     mir::optional_value<uint> clone_output_index;
      |                             ^
---
 include/qtmir/miral/display_configuration_storage.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/qtmir/miral/display_configuration_storage.h b/include/qtmir/miral/display_configuration_storage.h
index 91763d7c..8090877b 100644
--- a/include/qtmir/miral/display_configuration_storage.h
+++ b/include/qtmir/miral/display_configuration_storage.h
@@ -32,7 +32,7 @@ namespace miral
 struct DisplayConfigurationOptions
 {
     mir::optional_value<bool> used;
-    mir::optional_value<uint> clone_output_index;
+    mir::optional_value<unsigned int> clone_output_index;
     struct DisplayMode
     {
         mir::geometry::Size size;
-- 
2.31.1

